package/linux/application.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=0.1.0
+Encoding=UTF-8
+Name=_GCS
+Exec=_gcs
+TryExec=_gcs
+Comment=Configure, Tune, Diagnose, Track, & Upgrade FW for _ORG solutions
+Terminal=false
+Categories=Utility;Robotics;Engineering;
+Icon=_org
+Type=Application
```

Comment:
This good except I think it's better to not have the underscore so it matches what the default name for the gcs is (gcs), also the filename might better be called gcs.desktop